ofdb-app-clearance/src/page/index.rs

```diff
@@ -464,17 +464,17 @@ fn links_cs(lastrev: Option<&PlaceRevision>, currrev: &PlaceRevision) -> Changes
             homepage = l
                 .clone()
                 .and_then(|l| l.homepage)
-                .map(|h| h.into_string())
+                .map(Into::<String>::into)
```

Comment:
`.map(String::from)`?

Reply:
I just followed the advice of clippy ;-)